js/charts/rangefilter.js

```diff
@@ -83,7 +79,7 @@ function rangefilter() {
 
             var _filterParameters = filterParameters.get();
 
-            _filterParameters["data-range|" + dimension] = [dates[0], dates[1]];
```

Comment:
please replace this constant to filterParametersService

Reply:
@khushbu-maurya , can you please get back to @johibkhan2 regarding this?

Reply:
@admin-vizcentric its already done